scripts/release/rpm/test_mariner_in_docker.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 
-# This script should be run in a centos7 docker.
+# This script should be run in a Mariner 2.0 docker container.
```

Comment:
How about change `centos7` to `ubi or fedora` in `test_rpm_in_docker.sh`

Reply:
That‘s irrelevant to this PR. Let's keep this PR clean.